project/src/pages/main/main-screen.tsx

```diff
@@ -1,72 +1,52 @@
+import MainEmpty from 'components/main-empty/main-empty';
 
 type MainScreenProps = {
   offers: Offer[]
 }
 
+
 export default function MainScreen({offers}: MainScreenProps): JSX.Element {
+  const city = useAppSelector((state) => state.city);
+  const dispatch = useAppDispatch();
+  const changeCityHandler = (evt: MouseEvent<HTMLAnchorElement>) => {
+    evt.preventDefault();
+    const cityName = evt.currentTarget.innerText;
+    dispatch(changeCity(cityName));
+  };
+  const filteredOffers = offers.filter((offer) => offer.city.name === city);
 
   return (
     <div className="page page--gray page--main">
       <Header />
       <main className="page__main page__main--index">
         <h1 className="visually-hidden">Cities</h1>
-
-        <div className="tabs">
-          <section className="locations container">
-            <ul className="locations__list tabs__list">
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to={AppRoute.Main}>
-                  <span>Paris</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to={AppRoute.Main}>
-                  <span>Cologne</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to={AppRoute.Main}>
-                  <span>Brussels</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item tabs__item--active" to={AppRoute.Main}>
-                  <span>Amsterdam</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to={AppRoute.Main}>
-                  <span>Hamburg</span>
-                </Link>
-              </li>
-              <li className="locations__item">
-                <Link className="locations__item-link tabs__item" to={AppRoute.Main}>
-                  <span>Dusseldorf</span>
-                </Link>
-              </li>
-            </ul>
-          </section>
-        </div>
-
+        <Locations activeCity={city} onClick={changeCityHandler}/>
         <div className="cities">
-          <div className="cities__places-container container">
-            <section className="cities__places places">
-              <h2 className="visually-hidden">Places</h2>
-              <b className="places__found">312 places to stay in Amsterdam</b>
-              <Sorting />
-              <OffersList offers={offers}/>
-            </section>
-            <div className="cities__right-section">
-              <Map city={offers[0].city} offers={offers} mapClassName={MapClassNames.CITIES}/>
-            </div>
-          </div>
+          {filteredOffers.length ?
```

Comment:
Если будет 0 то выведется 0 
```suggestion
          {!!filteredOffers.length ?
          {filteredOffers.length > 0 ?
          {Boolean(filteredOffers.length) ?
```